packages/js/src/ui/Inbox.tsx

```diff
@@ -24,9 +28,11 @@ export const Inbox = (props: InboxProps) => {
   });
 
   return (
-    <AppearanceProvider elements={props.appearance?.elements} variables={props.appearance?.variables}>
-      <InternalInbox feeds={feeds()} />
-    </AppearanceProvider>
+    <LocalizationProvider localization={deepMerge(defaultLocalization, props.localization || {})}>
```

Comment:
✏ **suggestion:** I guess the `localization` value should be memoized, or this memoization should happen internally in the provider.